.changeset/hot-chicken-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
```

Comment:
```suggestion
"@primer/react": minor
```

I think we should do these as `minor` since we're adding in functionality to the package